docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
 
-* You can increase the map zoom by using the zoom in icon <img src="../img/button/zoom-in.jpg" class="ms-docbutton"/>.
+* **Increase** <img src="../img/button/zoom-in.jpg" class="ms-docbutton"/> the map zoom
 
-* You can decrease the map zoom by using the zoom out icon <img src="../img/button/zoom-out.jpg" class="ms-docbutton"/>.
+* **Decrease** <img src="../img/button/zoom-out.jpg" class="ms-docbutton"/> the map zoom
 
-* You can switch to full screen view <img src="../img/button/full-screen.jpg" class="ms-docbutton"/>.
+* **Switch to full screen**  <img src="../img/button/full-screen.jpg" class="ms-docbutton"/> view
 
-* You can navigate through the extents used in a session by going back <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> and forward <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> or to zoom to the maximum extent <img src="../img/button/max-extent.jpg" class="ms-docbutton"/>.
-
-3D Navigation
--------------
+* **Go back** <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> to the last change 
 
-Based on CesiumJS, a 3D Navigator is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows you to display the map contents in 3D mode, particularly the layers will be overlaid on the globe.
+* **Go forward** <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> to the last change 
 
-* **Open** a new map then **Add** a layer.
+* **Zoom to the maximum extent** <img src="../img/button/max-extent.jpg" class="ms-docbutton"/> the map
 
-* **Click** on the 3D icon <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> in the *Sidebar*.
+3D Navigation
+-------------
 
-The scene will switch from 2D to 3D display mode and you will be guided by a brief tutorial that explain you how to interact with the map.
+Based on CesiumJS, a **3D Navigator** <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows the user to display the map contents in 3D mode. In particularly the layers will be overlaid on the globe and it is possible to orbit around the globe trought the compass place in the upper right corner of the map.
 
 <img src="../img/side-bar/3D-mode.jpg" class="ms-docimage"/>
 
 ## Identify Tool
 
-The Identify tool  <img src="../img/button/identify.jpg" class="ms-docbutton"/> allows you to interact with the map by clicking on features to get information stored in the attribute table of the queried feature. It is active by default as the green color suggests.
-
-* **Add** a layer to the map (e.g. USA Population).
+Getting information about the layers, which have been loaded into the map, is possible thanks to the **Identify** button <img src="../img/button/identify.jpg" class="ms-docbutton"/>, through which the attribute table, where the characteristics of the layer are stored, is queried. 
```

Comment:
The  **Identify** tool <img src="../img/button/identify.jpg" class="ms-docbutton"/> allows to retrieve information about layers on the map.